packages/shared/lib/contexts/onboarding/actions/verifyAndStoreMnemonic.ts

```diff
@@ -0,0 +1,15 @@
+import { get } from 'svelte/store'
+
+import { storeMnemonic, verifyMnemonic } from '@core/profile-manager'
+
+import { onboardingProfile } from '../stores'
+
+/**
+ * Verifies, stores, then clears the mnemonic used in the onboarding flow.
+ */
+export async function verifyAndStoreMnemonic(): Promise<void> {
```

Comment:
I've taken care to clean it up after everywhere it's called, since I don't like the idea of including that side-effect in this function. I can create a higher level function that calls this one and then cleans it, which is currently in the component files. I can add a comment that makes note of that too :+1:

Reply:
I know you have added it, but someone else comes along might not and we don't want a mnemonic floating around by accident.